REMAINING.PUBLIC.md

```diff
@@ -0,0 +1,70 @@
+### Remaining public aliases
+
+annotation_reader
+cache.app_clearer
+$commandId
+debug.templating.engine.php
```

Comment:
the debug one does not need to be public IMO. You should never get it by its `debug.*` id (the `templating.engine.php` should stay public though)